valhalla/baldr/edge_elevation.h

```diff
@@ -0,0 +1,84 @@
+#ifndef VALHALLA_BALDR_EDGEELEVATION_H_
+#define VALHALLA_BALDR_EDGEELEVATION_H_
+
+namespace valhalla {
+namespace baldr {
+
+// Use elevation bins of 8 meters to store mean elevation. Clamp to a range
+// from -500 meters to 7683 meters.
+constexpr uint32_t kMaxStoredElevation = 1023;
+constexpr float kElevationBinSize = 8.0f;
```

Comment:
I will move it to just before spare, but that would still mean data incompatibility if we end up changing the precision at a future time. I can change to 13 (1m precision) if you want. I just wonder since this is "mean" elevation over the length of an entire edge I'm not sure how precise you really need. For now I'll change it to 12 bits (2m precision) and test. If you decide you want 1m then I will change to that.